Week1/Day3/ExercisesXPNinja/code.py

```diff
@@ -0,0 +1,41 @@
+companies = "Volkswagen, Toyota, Ford Motor, Honda, Chevrolet"
+companies_list = companies.split(",")
+print(f"{len(companies_list)} companies in the list")
+companies_list.sort(reverse=True)
+print(companies_list)
+
+companies_with_o = []
+for company in companies_list:
+    if 'o' in company.lower():
+        companies_with_o.append(company)
+
+print("Number of companies names with letter ‘o’:", len(companies_with_o))
+
+companies_without_i = []
+for company in companies_list:
+    if 'i' not in company.lower():
+        companies_without_i.append(company)
+
+print("Number of companies names without letter ‘i’:", len(companies_without_i))
+
+# Bonus
+new_companies = ["Honda", "Volkswagen", "Toyota", "Ford Motor", "Honda", "Chevrolet", "Toyota"]
+
+unique_companies_set = set(new_companies)
+
+unique_companies_list = list(unique_companies_set)
+
+unique_companies_list.sort()
+
+unique_companies_string = ", ".join(unique_companies_list)
+
```

Comment:
any idea to do all of the aboce in one line or in a different function

Reply:
unique_companies_list = list(set(new_companies))